sdaps_tools.py

```diff
@@ -1,13 +1,44 @@
+import subprocess
+import csv
 
-
-def setup_sdaps_project(tex_file, project_id):
+def setup_sdaps_project(tex_file,project_id):
     # TODO: Fill in code that sets up an SDAPS project for the given tex
     # file and with the given project id (=sdaps folder name).
-    pass
 
-def parse_image(image, project_id):
+    #The command for setting up sdaps project
+    create_project_command=['sdaps','setup', './assets/results/folder_name'+project_id,tex_file,'--add','./coins']
```

Comment:
`./coins` shouldn't be hardcoded. If our contest is going to use images that come in a folder, add an argument to `parse_image` that allows us to specify this folder.